conf/script/ha_setup

```diff
@@ -198,6 +199,21 @@ consul_watcher_failback() {
     fi
 }
 
+consul_watcher_stop_io() {
+    echo "Copy consul watcher files for data integrity check ..."
```

Comment:
Will be good to call out specific consul key being copied. This will help in debugging.

Reply:
Here we are copying the watcher file. This file contains the keyprefiix that will be matched to incoming IEM's keys.
@ujjwalpl , Should we print that prefix here ?

Reply:
Yes, here we copy the consul watcher configuration file.
@ujjwalpl Please elaborate.

Reply:
We should be printing ${SOURCE_HA_CONF}/consul_watcher_stop_io.json and /var/lib/hare/consul-server-c[1-2]-conf/

In general there should be no log statement without any values.

Reply:
Modified the log.
 @ujjwalpl , can you please take a look